lib/puppetserver/ca/cli.rb

```diff
@@ -121,6 +121,9 @@ def self.parse_general_inputs(inputs)
           opts.on('--version', 'Display the version') do |v|
             parsed['version'] = true
           end
+          opts.on('--config', 'Path to a custom puppet.conf') do |v|
```

Comment:
Was the `--config` option just totally ignored before?? Is there something else we need to do to ensure the custom config is used?

Reply:
......... weird... I did not mean to include this in my change.

Reply:
so.. are you saying this shouldn't be here? How does passing `--config` work now?

Reply:
Each action defines it.
At one point I thought that was dumb and started to do a maint to refactor it all into the cli class, but didn't finish it. Somehow when I checked out this branch some of that work came along?

Reply:
ahh, gotcha. so it sounds like this should be removed.